code/components/extra-natives-five/src/WeatherNatives.cpp

```diff
@@ -69,7 +69,7 @@ static HookFunction hookFunction([]()
 		hook::set_call(&getWeatherByName, location + 23);
 		hook::call(location + 23, getWeatherByNameHook);
 
-		g_currentWeather = hook::get_address<int*>(location + 30);
+		g_currentWeather = hook::get_address<int*>(location + (xbr::IsGameBuildOrGreater<3095>() ? 51 : 30));
```

Comment:
Are you sure the offset changed in 3095 and not in 3258? For me the old offset works fine in 3095.

Reply:
Sorry, I remembered that I had crashed on 3095 but it seems I was wrong, I just changed the version to 3258